### ISSUE TITLE
Submit button needs `disabled` attribute

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#268]
Button needs the `disabled` attribute so that people know that they can't submit the form.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592741-3936bfd4-9644-11e8-86aa-367d9d69ff30.png">
<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592742-39481770-9644-11e8-8cf5-8d195f97729f.png">

**Comment by srivama** [comment]
Please assign this to me.

**Comment by jagtalon** [comment]
Hey @srivama! Since no one is assigned to this, feel free to pick it up and make a PR! 🙌🏽

**Comment by srivama** [comment]
Cool. Thanks :smile:

**Comment by srivama** [comment]
The is-disabled attribute is already set in feedback-form.es6.js. Can you please elaborate on what needs to be changed? Also, do you have any newcomer setup guide and discussion channels?

**Comment by srivama** [comment]
Duckduckgo's slack channel isn't accepting users and the cardio token on the starter guide isn't working. How do I reach the community and fix my development environment on cardio?

**Comment by andrey-p** [comment]
Will be fixed in the next release.
